platform-ajax/src/test/java/com/softicar/platform/ajax/input/auto/complete/entity/AbstractAjaxAutoCompleteEntityTest.java

```diff
@@ -376,6 +368,11 @@ private void setExpectedServerValue(AjaxTestEntity serverValue) {
 			this.expectedServerValue = serverValue;
 		}
 
+		private void setExpectedServerValueExceptionMessage(IDisplayString exceptionMessage) {
```

Comment:
We can now assert the occurrence of exceptions (through exception messages). Previously, stuff never threw.

Reply:
very good